pkg/services/validation.go

```diff
@@ -44,23 +44,20 @@ func ValidateManifest(resType api.ResourceType, manifest datatypes.JSONMap) erro
 	switch resType {
 	case api.ResourceTypeSingle:
```

Comment:
Do we still need `ResourceTypeSingle`?

Reply:
Need more discussion about the needs of single api for restful. if no needed, will have a separated PR to handle.